src/components/CommandBar/CommandBar.tsx

```diff
@@ -149,6 +151,7 @@ export class CommandBar extends React.Component<ICommandBarProps, ICommandBarSta
                          data-command-key={ index }
                          aria-haspopup={ !!(item.items && item.items.length) }
                          role='menuitem'
+                         aria-label={ `${ item.name }. ${ index === 0 ? this.props.itemNavigationScreenReaderText : '' }` }
```

Comment:
The reason I did this I'm afraid the user overwrite the arialabel for the first item, and it doesn't read the command name anymore.